src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx

```diff
@@ -33,115 +33,188 @@ redirects:
+  plugins {
+    id("newrelic") version "${<AGENT_VERSION>}"
+  }
+}
+```
 
-    a. In your project’s root directory (`projectname/app`), add a `newrelic.properties` file with the following line:
+        </TabsPageItem>
+        <TabsPageItem id="project">
 
-      ```properties
-      com.newrelic.application_token=GENERATED_TOKEN
-      ```
-    b. Complete the configuration steps on [Configure ProGuard or DexGuard for Android apps](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps). Then come back to this page and move on to the next step. 
+You can also update your build file on the project-level:
 
+```
+plugins {
+  // for binary Gradle plugins that need to be resolved
+  id("newrelic") version "<AGENT_VERSION>" apply false
+}     
+```   
+        </TabsPageItem>
+    </TabsPages>
+</Tabs>
 
-3. Set app permissions: Ensure that your Android app requests `INTERNET` and `ACCESS_NETWORK_STATE` permissions by adding these lines to your `AndroidManifest.xml` file:
+    </Step>
+    <Step>
+        ### Apply the plugin on the app level
 
-  ```
-  <uses-permission android:name="android.permission.INTERNET" />
-  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-  ```
-4. To start New Relic's mobile monitoring agent: In your **Default Activity** (as defined in your **Manifest**), import the `NewRelic` class:
+Regardless of what route you took, you then need to apply the plugin on the app level:
 
-  ```
-  import com.newrelic.agent.android.NewRelic;
-  ```
+```
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+    id("newrelic")
+}
+```
 
-  <Callout variant="important">
-    We do not support starting the mobile monitoring agent in other classes, as that can cause unexpected or unstable behavior.
-  </Callout>
-5. To initialize mobile monitoring: In the `onCreate()` method, add this call:
+    </Step>
+</Steps>
 
-   ```java
-   NewRelic.withApplicationToken("GENERATED_TOKEN").start(this.getApplication());
-   ```
+For details on how to configure the behavior of the mobile monitoring agent plugin during Gradle builds, see the [New Relic Gradle plugin](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-new-relic-gradle-plugin) extension guide.
 
-6. Optional: Change the [logging level](/docs/mobile-monitoring/new-relic-mobile-android/api-guides/android-agent-configuration-feature-flags#logging).
-7. To build and run your application: Clean your project, then run your app in an emulator or device to generate traffic.
-8. Wait a few minutes, then view data for your Android app from the [**Summary** page](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-overview-page): Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > (select an app)**.
+## Set up ProGuard or DexGuard
```

Comment:
@cthomas-newrelic I'm thinking we can actually remove this step here and in the What's next? section link out to this file: https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps/

We can add the root directory instruction to the dexguard doc so info isn't lost. It's a confusing experience to have customers do a partial set up on one doc, then have them navigate to finish set up in a second. We'd much rather have this content all in one place, esp since not everyone uses these capabilities.